config/horizon.php

```diff
@@ -230,6 +229,20 @@
             'timeout' => 600, // NOTE timeout should always be at least several seconds shorter than the queue config's retry_after configuration value
             'nice' => 0,
         ],
+        'supervisor-3' => [
+            'connection' => 'redis',
+            'queue' => [
+                'scout',
+            ],
+            'balance' => 'simple',
+            'processes' => 1, // Fixed at exactly 1 process.
+            'maxTime' => 0,
+            'maxJobs' => 0,
+            'memory' => 128,
+            'tries' => 1,
+            'timeout' => 300, // NOTE timeout should always be at least several seconds shorter than the queue config's retry_after configuration value.
+            'nice' => 0,
+        ],
```

Comment:
Is there something I need to do beyond `sail artisan queue:restart` to pick up these changes? It seems like my local instance doesn't want to allocate any processes to the new supervisor.
![image](https://github.com/user-attachments/assets/84ad3d6f-5234-45fd-ba6b-cc84441882e8)

And there doesn't seem to be any tooltip or other way to see why it's got the red X.

Reply:
Very strange, this is what I see locally:

![Screenshot 2025-05-18 at 10 14 41 AM](https://github.com/user-attachments/assets/c2051e21-e55f-44d8-b5a1-bfac1716a75b)

The only thing I've done is `sail artisan horizon` - no other commands.

Something appears to be wrong with your local Horizon -- both supervisor-1 and supervisor-3 have "scout" queues. Try:
```
sail artisan horizon:terminate
sail artisan horizon:purge
sail artisan horizon
```

Reply:
That did it.